tests/index.test.js

```diff
@@ -0,0 +1,95 @@
+const request = require('supertest');
+const app = require('../index');
+const nock = require('nock');
+const mockSearchData = require('../mocks/searchData.json');
+const mockStylistData = require('../mocks/stylistData.json');
+const mockReviewData = require('../mocks/reviewData.json');
+const mockErrorData = require('../mocks/errorData.json');
+const mockErrorLocationData = require('../mocks/errorLocationData.json');
+const mockErrorValidationData = require('../mocks/errorValidationData.json');
+
+describe('GET results', () => {
+  it('should get search results based on a term and location', async () => {
+    const term = 'interlocks';
+    const location = 'boston';
+    nock('https://api.yelp.com')
```

Comment:
Nock intercepts this request that would've normally gone to an external API and instead mocks the response status code and data (200 and mockSearchData here, respectively).